detox/src/devices/drivers/android/tools/__mocks__/handles.js

```diff
@@ -1,22 +1,21 @@
-const DeviceHandle = require('../DeviceHandle');
-const EmulatorHandle = require('../EmulatorHandle');
-
 const mockAvdName = 'mockAvdName';
-class MockEmulatorHandle extends EmulatorHandle {
-  queryName() {
-    return Promise.resolve(mockAvdName);
-  }
-}
 
-const emulator5556 = new MockEmulatorHandle('emulator-5556\tdevice');
-const localhost5555 = new DeviceHandle('localhost:5555\tdevice');
-const ip5557 = new DeviceHandle('192.168.4.19:5557\tdevice');
-const deviceOffline = new MockEmulatorHandle('192.168.4.19:5559\toffline');
```

Comment:
I believe in pure mocking - avoiding the coupling of this scope with the logic of Handlers (namely, the format of the accepted string).

Reply:
🤦‍♂️ okay, whatever

Reply:
But that's a problem in both approaches

Reply:
Actually, it is not as sever. I'll reconsider this change.